packages/client/src/coordinator/connection/client.ts

```diff
@@ -73,6 +73,7 @@ export class StreamClient {
   rejectConnectionId?: Function;
   private connectionIdPromiseSafe?: SafePromise<string | undefined>;
   guestUserCreatePromise?: Promise<CreateGuestResponse>;
+  shouldRejectCallWhenBusy?: boolean;
```

Comment:
I don't see a reason why we keep this prop on the `StreamClient` instance.
Can you move it to `StreamVideoClient`?